app/assets/javascripts/analytics.js.erb

```diff
@@ -2,6 +2,25 @@
 var element = document.querySelector('[data-module="cookie-banner"]')
 new GOVUK.Modules.CookieBanner().start($(element))
 
+var redactPII = function() {
+  const params = new URLSearchParams(window.location.search)
+  for (const [key, value] of params.entries()) {
```

Comment:
We don't use `const` and `let`, only `var` as we don't transpile from ES6 and the new not supported in Internet Explorer (in v11 not supported in for-in and for-of loops and below v11 there is no support whatsoever).